### PR TITLE
Added term concussion issue #1030

### PR DESCRIPTION
**Pull request by kallia-p**
Added new term concussion (EFO:0011023) Subclass of head injury and Subclass of brain injury

### CODE REVIEW

**Comment by kallia-p** [comment]
@matentzn @dosumis I did a small edit in the latest commit (Added dbxrfs from NCIT and OMIT …67d5e39 to the newly added term concussion) and I resolved 2 conflicts see below (I followed the procedure for re-serialising the master and updating my branch from the master also pasted below at the very end). I have gone ahead in Atom and used the concussion edits for both conflicts. 

```
<<<<<<< HEAD
Declaration(Class(efo:EFO_0011023))
=======
Declaration(Class(efo:EFO_0011025))
>>>>>>> master


<<<<<<< HEAD
# Class: efo:EFO_0011023 (concussion)

AnnotationAssertion(Annotation(efo:definition_citation "https://en.wikipedia.org/wiki/Concussion") obo:IAO_0000115 efo:EFO_0011023 "A concussion, also known as a mild traumatic brain injury (mTBI), is a head injury that temporarily affects brain functioning. It is usually caused by a bump, blow or jolt to the head.")
AnnotationAssertion(obo:IAO_0000117 efo:EFO_0011023 "Elizabeth Langley")
AnnotationAssertion(efo:gwas_trait efo:EFO_0011023 "true"^^xsd:boolean)
AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "PMID:33017352")
AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "http://purl.obolibrary.org/obo/NCIT_C50502")
AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "http://purl.obolibrary.org/obo/OMIT_0003280")
AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011023 "brain concussion")
AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011023 "mTBI")
AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011023 "mild TBI")
AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011023 "mild traumatic brain injury")
AnnotationAssertion(rdfs:label efo:EFO_0011023 "concussion"@en)
SubClassOf(efo:EFO_0011023 obo:MONDO_0043510)
SubClassOf(efo:EFO_0011023 efo:EFO_0009505)
=======
# Class: efo:EFO_0011025 (10x 5' v1 sequencing)

AnnotationAssertion(obo:IAO_0000115 efo:EFO_0011025 "10x 5' v1 is the first version of the 10X sequencing technology that sequences from the 5' end of nucleic acid molecules. Differs from 3' v1 in the captured end from the polyadenylated transcript.")
AnnotationAssertion(dc:creator efo:EFO_0011025 "paola"^^xsd:string)
AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011025 "10X 5' v1")
AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011025 "10x 5' v1")
AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011025 "10x v1 5' sequencing")
AnnotationAssertion(oboInOwl:inSubset efo:EFO_0011025 efo:EFO_0010944)
AnnotationAssertion(rdfs:label efo:EFO_0011025 "10x 5' v1 sequencing")
SubClassOf(efo:EFO_0011025 efo:EFO_0009897)
>>>>>>> master
```

**Comment by kallia-p** [comment]
Resolving conflicts (and re-serialising master)
1. In Slack go to #efo notifications and message Lock EFO (required when making changes to the master, not required when just working on a branch) 
2. In Github Desktop change branch to master, fetch origin
3. In terminal cd to the src/ontology/ dir and run the command ../../bin robot convert --input efo-edit.owl --ofn --output efo-edit.owl (this will re-serialise the master file)
4. In Github Desktop commit to master (diff is large), push origin
5. Open branch
6. Update branch (Menu select Branch and click update from Master)
7. If there are conflicts open file in Atom and inspect. - Tip: Control + F === to find and move through the conflicts.

**Comment by matentzn** [comment]
In the diff, it seems yuo have lost `Class: efo:EFO_0011025 (10x 5' v1 sequencing)`? Was this by choice?

**Comment by kallia-p** [comment]
Not by choice and I am not sure how to go on about this now.
Thanks

**Comment by matentzn** [comment]
When you resolved the conflict, you were given two choices (master or head), but in this case, probably, neither was right - you probably have to merge the missing term back in. I would:

1. Find out with the master version of efo, in a text editor, where 
```
# Class: efo:EFO_0011025 (10x 5' v1 sequencing)
```

used to be

2. Copy the class with a text editor
3. Paste it back there in your branch 

The github diff should only show your added class!

Let me know if that works for you!

**Comment by dosumis** [comment]
This is a fake conflict caused by terms added with (near) consecutive ids hitting the same part of the file.  The correct solution in this case is just to remove the conflict marks as you want to keep edits coming from master and your branch.  You only need to choose one side vs the other if the same term or same line has been edited.  Even then you might want to manually reconcile.  However, real clashes are likely to be rare.

You are likely to see this quite often if URIGEN is assigning consecutive ids to terms added by different editors.

I think maybe the atom diff plugin is confusing matters here.

**Comment by matentzn** [comment]
Yeah I totally hate this problem. We have it in Mondo a lot!

File: src/ontology/efo-edit.owl [hunk 1]
```diff
@@ -203006,6 +203007,22 @@ AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011018 "BDNF")
 AnnotationAssertion(rdfs:label efo:EFO_0011018 "brain-derived neurotrophic factor measurement"@en)
 SubClassOf(efo:EFO_0011018 efo:EFO_0004747)
 
+# Class: efo:EFO_0011023 (concussion)
+
+AnnotationAssertion(Annotation(efo:definition_citation "https://en.wikipedia.org/wiki/Concussion") obo:IAO_0000115 efo:EFO_0011023 "A concussion, also known as a mild traumatic brain injury (mTBI), is a head injury that temporarily affects brain functioning. It is usually caused by a bump, blow or jolt to the head.")
+AnnotationAssertion(obo:IAO_0000117 efo:EFO_0011023 "Elizabeth Langley")
+AnnotationAssertion(efo:gwas_trait efo:EFO_0011023 "true"^^xsd:boolean)
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "PMID:33017352")
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "http://purl.obolibrary.org/obo/NCIT_C50502")
```
Comment (matentzn) [inline src/ontology/efo-edit.owl:203016]: Should be:
```
AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "NCIt:C50502")
```

to be consistent with other NCIT xrefs (note the lower case t in NCIt as well)

File: src/ontology/efo-edit.owl [hunk 2]
```diff
@@ -203006,6 +203007,22 @@ AnnotationAssertion(oboInOwl:hasExactSynonym efo:EFO_0011018 "BDNF")
...
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "http://purl.obolibrary.org/obo/OMIT_0003280")
```
Comment (matentzn) [inline src/ontology/efo-edit.owl:203017]:
```
AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0011023 "OMIT:0003280")
```

**Review by matentzn — APPROVED** [review]

**Pull request merged**